chain/chain/src/chain.rs

```diff
@@ -1080,7 +1080,7 @@ impl Chain {
         header: &BlockHeader,
         challenges: &mut Vec<ChallengeBody>,
     ) -> Result<(), Error> {
-        debug!(target: "chain", "Process block header: {} at {}", header.hash(), header.height());
+        debug!(target: "chain", block_hash=?header.hash(), height=header.height(), "process_block_header", );
```

Comment:
```suggestion
        debug!(target: "chain", block_hash=?header.hash(), height=header.height(), "process_block_header");
```

Unfortunately rustfmt can't really format some macro invocations, so we have to do it by hand... T_T